modules/ztp-worker-node-node-selector-compatibility.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+// Epic CNF-5335 (4.11), Story TELCODOCS-643
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-additional-worker-node-selector-comp"]
+= (Optional) Ensuring PTP and SR-IOV node selector compatibility
```

Comment:
Is this an optional procedure as well? I marked it as such for now, but please confirm

Reply:
I think it's not so optional. If a user wants to make sure ptp and sriov will work on the worker node, the node selector must be checked ind fixed if needed